django/db/migrations/graph.py

```diff
@@ -252,6 +252,12 @@ def __str__(self):
             sum(len(node.parents) for node in self.node_map.values()),
         )
 
+    def __repr__(self):
+        return '<MigrationGraph: nodes=%s, edges=%s>' % (
```

Comment:
Should use `self.__class__.__name__` instead of hardcoding the class name.
Also, could we factor the common code in `__str__` and `__repr__`?